packages/cashier/src/containers/cashier/cashier.tsx

```diff
@@ -249,6 +249,9 @@ const Cashier = observer(({ history, location, routes: routes_config }: TCashier
         return <Loading is_fullscreen />;
     }
 
+    // measure performance metrics (load cashier time)
+    setPerformanceValue('load_cashier_time');
```

Comment:
@sergei-deriv same question here

Reply:
@mahdiyeh-deriv the same answer :) when the loader is off and we need to show needed content I stop the timer